docs/python/python-tutorial.md

```diff
@@ -252,36 +253,33 @@ A best practice among Python developers is to avoid installing packages into a g
 
 > **Note**: For additional information about virtual environments, see [Environments](/docs/python/environments.md#creating-environments).
 
-1. Create and activate the virtual environment
+1. Create a Virtual Environment using the Create Environment Command
 
-   **Virtual environment creation for Windows**
+   From within VS Code, you can create non-global environments, using Venv or Anaconda, by opening the Command Palette (Ctrl+Shift+P), start typing the `Python: Create Environment` command to search, and then select the command. You can also trigger the `Python: Create Environment' command through the Getting Started with Python page.
 
-   ```bat
-   py -3 -m venv .venv
-   .venv\scripts\activate
-   ```
+   The command presents a list of environment types, Venv or Conda.
 
-   If the activate command generates the message "Activate.ps1 is not digitally signed. You cannot run this script on the
-   current system.", then you need to temporarily change the PowerShell execution policy to allow scripts to
-   run (see [About Execution Policies](https://go.microsoft.com/fwlink/?LinkID=135170) in the PowerShell documentation):
+   ![Create Environment dropdown](images/environments/create_environment_dropdown.png)
 
-   ```powershell
-   Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process
-   ```
+   If you are creating an environment using **Venv**, the command presents a list of interpreters that can be used for your project.
 
-   **Virtual environment creation for macOS/Linux**
+   ![Virtual environment interpreter selection](images/environments/interpreters-list.png)
 
-   ```bash
-   python3 -m venv .venv
-   source .venv/bin/activate
-   ```
-   >**Note**: When you create a new virtual environment, you should be prompted by VS Code to set it as the default for your workspace folder. If selected, the environment will automatically be activated when you open a new terminal.
+   If you are creating an environment using **Conda**, the command presents a list of Python versions that can be used for your project.
```

Comment:
Because this is a tutorial ,we can be opinionated and ask them to make a choice. For simplicity sake (and because this is a general script tutorial), I think we could suggest users to just choose `venv` and then for the rest of tutorial, assume that is what they're using

Reply:
Updated to reflect this suggestion